src/data/general-cs-quiz.ts

```diff
@@ -934,6 +934,24 @@ const generalCSQuiz = [
     Explanation: "Merge sort is a divide and conquer algorithm that has a best case time complexity of O(n*log n)",
     Link: "https://www.freecodecamp.org/news/an-intro-to-advanced-sorting-algorithms-merge-quick-radix-sort-in-javascript-b65842194597/",
   },
+  {
+    Question: "What is the advanatge of using linked lists over arrays?",
+    Answer: "Linked lists helps in faster insertions and deletions than arrays",
+    Distractor1: "Memory usage in linked lists is less than arrays",
+    Distractor2: "Reverse traversal is possible in singly linked lists",
+    Distractor3: "Random accessing elements is easier in linked lists than arrays",
+    Explanation: "In Linked lists there is no requirement to shift all other elements while doing insertions or deletions, only address gets updated.",
```

Comment:
Let's reword this so it is not as similar to the Geeks for Geeks explanation here
https://www.geeksforgeeks.org/advantages-and-disadvantages-of-linked-list/

```suggestion
    Explanation: "Linked lists have quicker insertions and deletions than arrays because there is no need to shift all other elements with lists.",
```